app/app/explore/page.tsx

```diff
@@ -0,0 +1,280 @@
+        ]);
+        setPools(poolsData);
+        setSavers(Object.values(saversData).map(s => s.savers));
+        setRunePriceUSD(parseFloat(statsData.runePriceUSD));
+      } catch (error) {
+        console.error('Error fetching data:', error);
+      } finally {
+        setLoading(false);
+      }
+    };
+
+    fetchData();
+  }, []);
+
+  const getAssetSymbol = (asset: string) => {
+    return asset.split(".")[1].split("-")[0] || asset;
+  };
+
+  const getLogoPath = (asset: string) => {
+    const assetLower = asset.toLowerCase();
+    return `https://storage.googleapis.com/token-list-swapkit-dev/images/${assetLower}.png`;
+  };
+
+  const calculateTVL = (pool: Pool) => {
+    const assetValueInUSD = parseFloat(pool.assetDepth) * parseFloat(pool.assetPriceUSD) / 1e8;
+    const runeValueInUSD = parseFloat(pool.runeDepth) * runePriceUSD / 1e8;
+    return (assetValueInUSD + runeValueInUSD) / 1e6;
+  };
+
+  const calculateVolumeUSD = (pool: Pool) => {
+    const volumeInRune = parseFloat(pool.volume24h) / 1e8;
+    return volumeInRune * runePriceUSD;
+  };
+
+  const calculateVolumeDepthRatio = (pool: Pool) => {
+    const volumeUSD = calculateVolumeUSD(pool);
+    const tvlUSD = calculateTVL(pool) * 1e6;
+    return volumeUSD / tvlUSD;
+  };
+
+  const sortData = (key: string) => {
+   let direction = 'asc';
+   if (sortConfig.key === key && sortConfig.direction === 'asc') {
+     direction = 'desc';
+   }
+   setSortConfig({ key, direction });
+
+   const sortedData = [...(activeTab === 'liquidity' ? pools : savers)].sort((a, b) => {
+     if (key === 'tvl') {
+       const aTvl = activeTab === 'liquidity' ? calculateTVL(a as Pool) : parseFloat(a.saversDepth) * parseFloat(a.assetPriceUSD) / 1e14;
+       const bTvl = activeTab === 'liquidity' ? calculateTVL(b as Pool) : parseFloat(b.saversDepth) * parseFloat(b.assetPriceUSD) / 1e14;
+       return direction === 'asc' ? aTvl - bTvl : bTvl - aTvl;
+     } else if (key === 'apr') {
+       const aApr = activeTab === 'liquidity' ? parseFloat((a as Pool).poolAPY) : parseFloat(a.saversReturn);
+       const bApr = activeTab === 'liquidity' ? parseFloat((b as Pool).poolAPY) : parseFloat(b.saversReturn);
+       return direction === 'asc' ? aApr - bApr : bApr - aApr;
+     }
+     return 0;
+   });
+
+   if (activeTab === 'liquidity') {
+     setPools(sortedData as Pool[]);
+   } else {
+     setSavers(sortedData as Saver[]);
+   }
+ };
+
+  const topPools = pools.slice(0, 3);
+  const topSavers = savers.slice(0, 3);
+
+  return (
+    <>
+      <main className="mx-auto">
+        <nav className="mb-4">
+          <ul className="flex space-x-8 font-semibold font-gt-america-exp text-2xl">
+            <li>
+              <a
+                href="#"
+                onClick={() => setActiveTab('liquidity')}
+                className={`pb-2 no-underline ${activeTab === 'liquidity' ? 'text-gray-900 font-bold' : 'text-gray-400 hover:text-gray-700 font-medium'}`}
```

Comment:
should use twmerge and break this out into a separate component such that the active state is  agnostic to the active tab.